app/frontend/src/components/CommunityGuidelines/CommunityGuidelines.tsx

```diff
@@ -58,6 +62,15 @@ export default function CommunityGuidelines({
   const [completed, setCompleted] = useSafeState(isMounted, false);
   const [error, setError] = useState("");
```

Comment:
```suggestion
```
This is probably redundant since you are now using React Query to fetch stuff, which has error state built in already.

Reply:
It uses react query to fetch the guidelines; but not to submit the form. This component is used in both signup and in jail. In signup it doesn't do an API call, I don't think.